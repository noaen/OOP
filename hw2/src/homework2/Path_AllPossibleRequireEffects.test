# Sample Testing file

# Create a graph
CreateGraph graph1

# Create nodes
CreateNode n1 1
CreateNode n2 2
CreateNode n3 3

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add an edge
AddEdge graph1 n1 n2
AddEdge graph1 n1 n3
AddEdge graph1 n2 n3

# find path
FindPath graph1 n1 -> n2
FindPath graph1 n1 -> n3
FindPath graph1 n1 -> n2 n3
FindPath graph1 n1 -> n1 n2 n3
FindPath graph1 n1 n2 -> n1 n2 n3
FindPath graph1 n1 n2 -> n1 n2
FindPath graph1 n1 n2 -> n1
FindPath graph1 n1 n2 n3 -> n1 n2 n3
FindPath graph1 n1 n2 n3 -> n2 n3
