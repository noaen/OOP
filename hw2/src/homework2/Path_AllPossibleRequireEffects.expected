# Sample Testing file

# Create a graph
created graph graph1

# Create nodes
created node n1 with cost 1
created node n2 with cost 2
created node n3 with cost 3

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add an edge
added edge from n1 to n2 in graph1
added edge from n1 to n3 in graph1
added edge from n2 to n3 in graph1

# find path
found path in graph1: n1 n3 n2 with cost 3
found path in graph1: n1 n3 with cost 2
found path in graph1: n1 n3 with cost 2
found path in graph1: n1 with cost 1
found path in graph1: n1 with cost 1
found path in graph1: n1 with cost 1
found path in graph1: n1 with cost 1
found path in graph1: n1 with cost 1
found path in graph1: n2 with cost 1
